app/controllers/publishers/vacancies/feedbacks_controller.rb

```diff
@@ -1,13 +1,18 @@
 class Publishers::Vacancies::FeedbacksController < Publishers::Vacancies::BaseController
+  def new
+    @vacancy = VacancyPresenter.new(Vacancy.published.find(params[:job_id]))
+    @feedback_form = Publishers::JobListing::FeedbackForm.new
+  end
+
   def create
     @vacancy = VacancyPresenter.new(Vacancy.published.find(params[:job_id]))
```

Comment:
Same here, no need to set `@vacancy`

Below in `feedback_attributes` change `@vacancy` to `vacancy`